float64[] pose0
float64[] levels
dwal_planner/SampledPath[] paths
